matter.py

```diff
@@ -325,11 +325,11 @@ def prepare_source_dir():
     if image:
         if not exists(image):
             error(f"{image} does not exist")
-        if os.path.splitext(image)[1] not in (".png", ".tga", ".jpg", ".jpeg"):
-            error(f"Background image must be one of .png, .tga, .jpg or .jpeg formats.")
+        if os.path.splitext(image)[1] not in (".png", ".jpg", ".jpeg", ".tga"):
+            error("Background image must be one of .png, .jpg, .jpeg or .tga formats.")
         image_name = basename(image)
         copyfile(image, f"{INSTALLATION_SOURCE_DIR}/{image_name}")
-        if background:
+        if user_args.background:
```

Comment:
This condition seems good, but you also changed `background` for some reason even though it is not used after this. Is there a reason I'm not seeing?

Reply:
That's because the background variable has been set to either a user-defined value or a default value, so we will need to check user_args.background instead to determine whether the script was called with a --background value.